.github/workflows/benchmark.yml

```diff
@@ -26,9 +38,28 @@ jobs:
     runs-on: "ubuntu-22.04" # _SLANG_DEV_CONTAINER_BASE_IMAGE_ (keep in sync)
 
     # Only run on the main repo (not forks), unless it is a dry run:
-    if: "${{ github.repository == 'NomicFoundation/slang' || inputs.dryRun == true }}"
+    if: "${{ github.repository == 'NomicFoundation/slang' || github.event.inputs.dryRun == true }}"
 
     steps:
+      # Installs basic dependencies when running locally with https://github.com/nektos/act,
```

Comment:
I wonder why ACT is needed?
IIUC, we can always run `bencher`/submit new reports locally.

Reply:
It isn't really needed, though I must run the test in a virtual environment: I'm in macOS and the tools don't work here. I can run it inside devcontainers, within vscode, but that makes my whole ~IDE~ system to collapse. That's why I thought of ACT.

Reply:
Happy to discuss alternatives though.